__device-tests__/gutenberg-editor-sanity-test-4-visual.test.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * Internal dependencies
+ */
+const { blockNames } = editorPage;
+const { toggleOrientation, selectTextFromTextInput, setClipboard } = e2eUtils;
+import { takeScreenshot } from './utils';
+
+describe( 'Gutenberg Editor - Test Suite 4', () => {
+	it( 'Spacer block - Spacer in horizontal layout works as expected', async () => {
+		await editorPage.addNewBlock( blockNames.spacer );
+
+		await toggleOrientation( editorPage.driver );
+
+		// Wait for the device to finish rotating
+		await editorPage.driver.sleep( 3000 );
```

Comment:
Would it make sense to poll `getOrientation` on an interval or does that method prematurely return a value that does not match reality?

Reply:
It returns the return value quite fast since the rotation doesn't take much time but in some cases, some UI elements haven't finished getting to their positions resulting in having them in the wrong places when the screenshot is taken 😅

Reply:
👍🏻 I figured that was the case. Thank you for confirming.